content/altinn-studio/app-creation/configuration/eformidling/_index.md

```diff
@@ -0,0 +1,186 @@
+    stream.Position = 0;
+    StreamContent streamContent = new StreamContent(stream);
+    streamContent.Headers.ContentType = MediaTypeHeaderValue.Parse("application/xml");
+    return await Task.FromResult(("arkivmelding.xml", stream));
+}
+```
+
+## Sette mottaker for forsendelse i applogikken
+
+I App.cs kan man overstyre metoden som henter ut mottaker av forsendelsen fra `applicationmetadata.json`.
+Denne funksjonaliteten kan benyttes dersom dataen skal sendes til ulike mottakere basert på input fra slutbruker i skjemaet. 
+
+Det må tre steg til for å sette mottaker i applogikken, og alle endringer gjøres i `App.cs` finnes i repoet under `App/logic`
+
+1. Øverst i filen må det legges til en referanse til eFormidlings biblioteket. 
+  `using Altinn.Common.EFormidlingClient.Models.SBD;` 
+
+2. Legg til funksjonen nedenfor i klassen. 
+   Forventet output fra denne metoden er en lite som inneholder minst ett receiver-objekt.
+
+    ```cs
+    public override async Task<List<Receiver>> GetEFormidlingReceivers(Instance instance)
+    {    
+        Identifier identifier = new Identifier
+        {
+            Authority = "iso6523-actorid-upis"
+        };
+
+        // 0192 prefix for all Norwegian organisations.
+        identifier.Value = "[INSERT ORGANISATION NUMBER HERE WITH PREFIX `0192:`]" ; 
+
+        Receiver receiver = new Receiver { Identifier = identifier };
+        return new List<Receiver> { receiver };
+    }
+    ```
+
+3. Legg til egen logikk for å populere `identifier.Value` i fiksjonen.
+   Merk at det kun er norske organisasjonsnummer som støttes som mottaker per April 2021, 
+   og at prefiksen `0192:` er påkrevd før organisasjonsnummeret.
+
+
+## Kjøring med eFormidling mock lokalt 
+
+For å kjøre løsningen lokalt må man hente nødvendige filer fra 2 forskjellige repository
+1. https://github.com/felleslosninger/efm-mocks.git (Selve mock løsningen for å kunne kjøre eFormidling lokalt)
+2. https://github.com/Altinn/altinn-studio/tree/master/src/development (Konfigurasjon for å kunne kjøre hele løsningen enklere med docker compose)
+
+
+Hent ned mockløsningen lokalt for å teste forsendelser med eFormidling:
+```cmd
+git clone --branch development https://github.com/felleslosninger/efm-mocks.git
+```
+
+Pr d.d. Mars 2021, så er det anbefalt å bruke development branch ettersom det foreligger feil i master branch.
+For mer detaljert oppsett se README i repoet: 
+https://github.com/Altinn/altinn-studio/blob/master/src/Altinn.Common/Altinn.EFormidlingClient/Altinn.EFormidlingClient/README.md
+
+Det er 2 måter å kjøre mock løsningen med integrasjonspunktet lokalt: Integrasjonspunktet kjører seperat som en jar-fil eller så kjører hele løsningen i docker vha docker compose.
+
+•	(Foretrukket metode) Kopier docker compose filen under src/development/EFormidlingMock i Altinn Studio repo (https://github.com/Altinn/altinn-studio/tree/master/src/development), og "integrationpoint" mappen som inneholder konfigurasjon for løsningen. 
```

Comment:
• ? Should be list?

Reply:
Probably. The stepts didn't quite make sense when Jeeva tested them, so we'll try re-writing it.

Reply:
Re-written the section on mock solution locally